core/windows_and_tabs/tabs_win.py

```diff
@@ -0,0 +1,15 @@
+from talon import Context, actions
+
+ctx = Context()
+ctx.matches = r"""
+os: windows
+"""
+
+
+@ctx.action_class("user")
+class UserActions:
+    def tab_move_left():
+        actions.key("ctrl-shift-pageup")
+
+    def tab_move_right():
+        actions.key("ctrl-shift-pagedown")
```

Comment:
These commands at least work in the Mac version of Microsoft Edge so would recommend you implement this cross platform.